### DISCUSSION TITLE
Will DRF support Django version 5.0 latest release?

### DISCUSSION THREAD

**Discussion by steven-kamanga** [encode/django-rest-framework#9178] [Question & Answer]
With the latest release of Django 5.0, are there any concerns on the compatibility of the Rest framework worth Django 5.0. If any please state them below.

**Comment by rob4226** [comment]
Here are a couple of things:

1. In Django 5.0, GET request for logout was removed (only POST supported now) and it seems like in DRF we still do GET request to the endpoint. #9206 

    > Support for logging out via GET requests in the `django.contrib.auth.views.LogoutView` and 
`django.contrib.auth.views.logout_then_login()` is removed. Per [Features removed in 5.0](https://docs.djangoproject.com/en/5.0/releases/5.0/#features-removed-in-5-0)

    Appears to be handled in PR #9208

2. [Database-computed default values](https://docs.djangoproject.com/en/5.0/releases/5.0/#database-computed-default-values) Django 5.0 introduces `db_default`, this now raises the question: should DRF be aware of fields with db_default set? #9257 

3. [Database generated model field](https://docs.djangoproject.com/en/5.0/releases/5.0/#database-generated-model-field) The new [GeneratedField](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.GeneratedField) allows creation of database generated columns. This field can be used on all supported database backends to create a field that is always computed from other fields.  
**Does anything need to be done in DRF to support this new model field?**

---

[Django 5.0 release notes](https://docs.djangoproject.com/en/5.0/releases/5.0/#django-5-0-release-notes)

**Reply by igorgaming** [reply]
4. [repr() on fields with gettext_lazy labels](https://github.com/encode/django-rest-framework/discussions/9259) https://github.com/encode/django-rest-framework/pull/9278
